lib/Doctrine/Common/NotifyPropertyChanged.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Doctrine\Common;
+
+/**
+ * Contract for classes that provide the service of notifying listeners of
```

Comment:
This description needs improvement: currently, you are replicating the name of the class in it. 

Also try to read it by assuming no previous knowledge about it

Reply:
> What would you recommend as a description for this interface?

That's what I asked you to come up with ;-)

Reply:
Suggestion:

> Interface for classes that notify event listeners of changes to their managed properties.
>
> This interface is implemented by objects that manually want to notify their object manager or other listeners when properties changes, instead of relying on the object manager to compute property changes itself when changes are to be persisted.